validator/sawtooth_validator/networking/dispatch.py

```diff
@@ -202,6 +202,10 @@ def _determine_next(self, message_id, future):
             LOGGER.exception("Dispatcher timeout waiting on handler result.")
             raise
 
+        if res is None:
+            LOGGER.debug('Ignoring result previous error')
```

Comment:
This logging statement should be clearer and include more information as to the context.